react/features/face-landmarks/functions.js

```diff
@@ -110,7 +110,7 @@ export async function sendFacialExpressionsWebhook(state: Object) {
         meetingFqn: extractFqnFromPath(),
         sessionId: conference.sessionId,
         submitted: Date.now(),
-        emotions: facialExpressionsBuffer,
+        emotions: faceExpressionsBuffer,
```

Comment:
Can we consolidate in "expressions" here?

Reply:
looks like a backend change